website/content/docs/secrets/ad/migration-guide.mdx

```diff
@@ -109,6 +109,14 @@ the chance of this happening. Additionally, tuning the AD secrets engine [last_r
 parameter could help mitigate applications reading stale passwords, since the parameter allows
 rotation of the password if it's been rotated out-of-band within a given duration.
 
+##### Lazy vs Auto password rotation
+
+~> **Note**: The AD secrets engine "lazy rotates" the passwords on the condition that both
+the rotation period has been exceeded _AND_ the engine is receives a request for the role.
+In contrast, the LDAP secret engine automatically rotates the passwords when the
+`rotation_period` is met (client requests not required).  This difference in behaviour might
+need to be taken into account by consuming clients.
```

Comment:
```suggestion

<Note title="Lazy rotation vs automatic rotation">

  The AD secrets engine uses **lazy rotation** for passwords. With lazy
  rotation, passwords rotate whenever the engine receives a request for a role
  whose rotation period has elapsed.
  
  The LDAP secret engine uses **automatic rotation** for passwords. With
  automatic rotation, passwords are rotated as soon as the rotation period
  elapses, without waiting for a client request.

  Clients may need to allow for longer response times when contacting engines
  that use "lazy" rotation.

</Note>
```

Style correction: write in active voice, avoid italics in favor of bolding, avoid "this" as a pronoun